docs/navigation-prop.md

```diff
@@ -268,3 +298,25 @@ const navigateAction = NavigationActions.navigate({
 });
 this.props.navigation.dispatch(navigateAction);
 ```
+
+### `dangerouslyGetParent` - get parent navigator
```

Comment:
another good use case for this is to find the index of the route in the parents route list, so in the case of a stack if you are at index 0 then you may not want to render a back button, but if you're somewhere else in the routes then you will. related to https://github.com/react-navigation/rfcs/issues/57